feature-configs/deploy/container-mount-namespace/container-private-mounts/mc-container-mount-namespace.yaml

```diff
@@ -1,7 +1,9 @@
 apiVersion: machineconfiguration.openshift.io/v1
 kind: MachineConfig
 metadata:
-  name: container-mount-namespace
+  name: container-mount-namespace-and-kubelet-conf
```

Comment:
I'm actually surprised that the name change at this location didn't break the CI lanes.  We could just leave this file unchanged, but it's okay as long as it didn't break cnf-tests.

Reply:
I thought the same thing. The CI lane looks for a machine config with the container-mount-namespace unit in it. The specific name of the MC isn't checked.